help/en/html/hardware/cmri/CMRI.shtml

```diff
@@ -961,7 +968,8 @@ for easy access click an item in this list:</p>
 
   <p><img alt="" src="images/cmrinetinit-SMINI.png" width="427"
   height="169"></p>
-
+</ul>
```

Comment:
See comments above about allowed list contents.

Reply:
Corrected lines in error.